SPECS/xenopsd.spec

```diff
@@ -95,20 +114,54 @@ cp %{SOURCE6} xenopsd-network-conf
 cp %{SOURCE7} xenopsd-64-conf
 
 %build
+# this is a hack: we build and install two builds into the source
+# directory under root1 and root2. In the install step all we do is
+# copying files from it.
+
+mkdir root1 root2
 ./configure --libexecdir %{_libexecdir}/%{name}
+
+# regular build
+make
+make install DESTDIR=$PWD/root1 LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
+make clean
+
+# now build for coverage profiling
+make coverage
 make
+make install DESTDIR=$PWD/root2 LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
 
 %install
-make install DESTDIR=%{buildroot} LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
+# this installs the files from the first build
+rsync -a root1/ %{buildroot} 
```

Comment:
Please use cp with the appropriate flags (or maybe install), don't add an unnecessary dependency on rsync.